understanding/21/text-spacing.html

```diff
@@ -23,7 +23,9 @@ <h2>Intent</h2>
          <p>This SC focuses on the adaptability of content to an increase in spacing between lines, words, letters, and paragraphs. Any combination of these may assist a user with effectively reading text. As well, ensuring that content correctly adapts when users override author settings for spacing also significantly increases the likelihood other style preferences can be set by the user. For example, a user may need to change to a wider font family than the author has set in order to effectively read text. </p>
       	<section>
          <h3 id="authorresp">Author Responsibility </h3>
-         <p>This SC does not dictate that authors must set all their content to the specified metrics. Rather, it specifies that an author's content has the ability to be set to those metrics without loss of content or functionality. The author requirement is both to not interfere with a user's ability to override the author settings, and to ensure that content thus modified does not break content in the manners shown in figures 1 through 3 in Effects of Not Allowing for Spacing Override. The values in the SC are a baseline. Authors are encouraged to surpass the values specified, not see them as a ceiling to build to. If the user chooses to go beyond the metrics specified any resulting loss of content or functionality is the user's responsibility.</p>
+         <p>This SC does not dictate that authors must set all their content to the specified metrics, nor does the SC intend to imply that all users will adjust the specified metrics. Rather, it specifies that should a user choose to set any of these metrics they can do so without any loss of content or functionality. The author requirement is both to not interfere with a user's ability to override the author settings, and to ensure that content thus modified does not break content in the manners shown in figures 1 through 3 in Effects of Not Allowing for Spacing Override.</p>
+         <p>These metrics are not meant to represent minimum requirements to adhere to, as some metrics may not even be applicable to certain languages. For example, languages such as Japanese do not use spacing following paragraphs, meaning that users are unlikely to make any paragraph spacing changes in practice. If the user chooses to go beyond the metrics specified any resulting loss of content or functionality is the user's responsibility. It is benficial for users if authors use any locally available guidance for improving readability in the local language or writing system.</p>
```

Comment:
"These metrics are not meant to represent minimum requirements to adhere to" is just incorrect; that's *exactly* what these metrics are meant to represent, it's just that some of them are inapplicable in some circumstances.

```suggestion
         <p>In some human languages and scripts, some of the metrics specified by the SC are inapplicable. For example, languages such as Japanese do not use spacing following paragraphs, meaning that users are unlikely to make any paragraph spacing changes in practice. The exception in this SC allows authors to ignore text style properties which are inapplicable to the combination of language and script being used.</p>
         <p>If the user chooses to go beyond the metrics specified, any resulting loss of content or functionality is the user's responsibility. It is beneficial for users if authors use any locally available guidance for improving readability in the local language or writing system.</p>
```